keyboards/gboards/k/ginny/keymaps/default/user.def

```diff
@@ -0,0 +1,48 @@
+/* Welcome to ASETNIOP
+ *
+ * This file contains your personal chords, the  basic layout, symbols and 
+ * layers are in different files. Refer to dicts.def for their locations!
+ *
+ * This is a very different layout based off this table
+ * http://asetniop.com/ 
+ *
+ * Unlike standard Asetniop there are a few concessions that must be
+ * made. Namely we don't do partials and if a chord has multiple left/
+ * right words we go with the most common of the two.
+ *
+ * Also note, All Combos _must_ be done through Command Mode! If you
+ * use a combo enough add it to your dictionary in the USR section!
+ *
+ * Two 'layers' are avalible, CMD and USR. Just add these into a chord. 
```

Comment:
`avalible` -> `available` (multiple times in this file)

Reply:
Fixed the spelling issues and redid the perms.